core/field.js

```diff
@@ -124,6 +129,20 @@ Blockly.Field.cacheReference_ = 0;
  */
 Blockly.Field.BORDER_RECT_DEFAULT_HEIGHT = 16;
 
+/**
+ * The default height of the text element on any field.
+ * @type {number}
+ * @package
+ */
+Blockly.Field.TEXT_DEFAULT_HEIGHT = 12.5;
+
```

Comment:
What is this number based on? Does it have any relationship with Blockly.FieldText.FONTSIZE?

Reply:
ie: if a developer changes the font size from 11pt to 12pt, would they have to fiddle around with this number?

Reply:
Yes, they would have to fiddle with it. As mentioned in one of the other bugs, we don't handle font size changes correctly.

Right now this just matches the previous value, though it often showed up as an implicit part of another sizing term.